configure.ac

```diff
@@ -357,6 +357,11 @@ AC_CHECK_HEADER([X11/extensions/Xrandr.h], [],
 
 CFLAGS="$save_CFLAGS"
 
+# checking for libcheck (used for unit testing)
+PKG_CHECK_MODULES([CHECK], [check >= 0.10.0])
+AC_CHECK_HEADER([check.h], [],
+  [AC_MSG_ERROR([please install check])])
```

Comment:
I noticed that this makes `check` is always required to build xrdp even when users try to install xrdp to their system. I don't think unit testing is always required. Performing unit tests during CI is enough. What do you think?

Reply:
Yes, I think that's a great idea. I see you've raised #1860 - I'll take a look.